src/configManager/models/configHandler.js

```diff
@@ -0,0 +1,88 @@
+const dbConnector = require('./database/databaseConnector');
+const constConfig = require('../../common/consts').CONFIG;
+const logger = require('../../common/logger');
+
```

Comment:
first of all , I expect to see just two export- what are used by controller - updateConfig, getConfig